framework/include/utils/MathFVUtils.h

```diff
@@ -197,21 +191,14 @@ interpCoeffs(const InterpMethod m,
   switch (m)
   {
     case InterpMethod::Average:
+    case InterpMethod::SkewCorrectedAverage:
```

Comment:
The interpolation methods are like squashing limiter type and `apply_gradient_to_skewness` together. We have user code that relies on `InterpMethod == SkewCorrectedAverage` in order to determine whether to apply gradient corrections to skewness and they may pass that enumeration to these functions so we must be sure we can handle that enumeration. In the future it sounds like @grmnptr thinks that the idea of skew correction and the idea of central-differencing/second-order-upwind/etc. can be separated ... so then there may no longer be an enumeration option like `SkewCorrectedAverage` ... instead just `Average` again ... at which point perhaps we can abolish the interpolation methods altogether since we will have a 1-to-1 correspondence to the limiter types

Reply:
Yes, I'm planning to add to/modify these things in the near future.